src/features/apiKeys/apiKeys.tsx

```
@@ -0,0 +1,119 @@
+import type { ApiKeyTableData } from "@/models/apiKeys"
+import apiKeysService from "@/services/apiKeysService"
+import type { SortingState } from "@tanstack/react-table"
+import { Row } from "@tanstack/react-table"
+import { ApiKeyCreateDialog } from "@/components/custom/system/ApiKeyCreateDialog"
+import { toast } from "@/hooks/use-toast"
+import { PlusIcon } from "lucide-react"
+import { Button } from "@/components/ui/button"
+import Spinner from "@/components/custom/LoadingOverlay"
+import { ConfirmationDialogActionType } from "@/components/confirmationDialogAction"
+
+export interface ApiKeysPageProps {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)